### PR TITLE
pytdxmeasure: update to new confidential computing measurement protocol

### PR DESCRIPTION
**Pull request by jialeif**
New ACPI device changed to /sys/firmware/acpi/tables/CCEL
Add 2 fields in event log, CC type and CC subtype.
Please refer to section 4.3.3 CC-Event Log in GHCI specification for TDX 1.5

Signed-off-by: jialeie <jialei.feng@intel.com>

### CODE REVIEW

File: utils/pytdxmeasure/pytdxmeasure/ccel.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 LOG = logging.getLogger(__name__)
 
 
-class TDEL(BinaryBlob):
+class CCEL(BinaryBlob):
     """
-    Manage th TDEL ACPI table. It should run within TD guest.
+    Manage th CCEL ACPI table. It should run within TD guest.
```
Comment (jurobystricky) [inline utils/pytdxmeasure/pytdxmeasure/ccel.py:19]: th -> the

File: utils/pytdxmeasure/pytdxmeasure/ccel.py [hunk 2]
```diff
@@ -43,6 +43,22 @@ def oem_id(self):
...
+        return cc_type
+
+    @property
+    def cc_subtype(self):
+        """
+        Confidential Computing specific sub-type in integer
+        """
+        cc_subtype, _ = self.get_uint8(37)
+        return cc_subtype
+
```
Comment (jurobystricky) [inline utils/pytdxmeasure/pytdxmeasure/ccel.py:61]: 36, 37: Is it documented somewhere?

**Review by jurobystricky — COMMENTED** [review]
Just non-functional nitpicks.

**Comment by jialeif** [comment]
> Just non-functional nitpicks.

It's very detailed review~ Thanks for improving my PR

File: utils/pytdxmeasure/pytdxmeasure/ccel.py (hunk 2 above)
Comment (jialeif) [reply utils/pytdxmeasure/pytdxmeasure/ccel.py:61]: I add more info in commit msg: "Please refer to section 4.3.3 CC-Event Log in GHCI specification for TDX 1.5"

File: utils/pytdxmeasure/pytdxmeasure/ccel.py (hunk 1 above)
Comment (jialeif) [reply utils/pytdxmeasure/pytdxmeasure/ccel.py]: done

**Review by kenplusplus — APPROVED** [review]
LGTM

**Pull request merged**